.github/workflows/ci.yml

```diff
@@ -93,4 +93,25 @@ jobs:
             rm diff.txt || true # we don't fail if the file does not exist
           done;
           exit $code
+  make-httpd-maintainer-mode:
+    runs-on: ubuntu-latest
+    name: "Build with httpd in maintainer-mode"
+    container:
+      image: quay.io/vchlup/httpd-dev:latest
```

Comment:
Update for: https://quay.io/repository/mod_cluster/ci-httpd-dev

Reply:
I did prefix this with "ci-" so that once we have images for general consumptions aren't confused with internal images - but let me know if there's a better name.